### DISCUSSION TITLE
Disable Sanitized Graphql Query

### DISCUSSION THREAD

**Discussion by alvin562** [open-telemetry/opentelemetry-java-instrumentation#6425] [Q&A]
Is there an property/flag to disable sanitization of graphql queries within spans?  Something similar to the **otel.instrumentation.common.db-statement-sanitizer.enabled** for databases but for graphql queries.  

Relevant link:

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/instrumentation/graphql.md

**Comment by trask** [comment]
hi @alvin562! check out https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/graphql-java-12.0/javaagent/README.md
